### PR TITLE
created the function

### PR DESCRIPTION
**Pull request by hasan356**

### CODE REVIEW

File: downloader.sh [hunk 1]
```diff
@@ -1,6 +1,13 @@
 a="$#"
+
 declare -i chaps=0,chape=0,t=0,c=0,end_loop=0,d1=0,d2=0
 #checking for existence of -t , --chap in argument
+function()
+{
+local link=$1
+local start_end="$(echo $link | grep -aob '"' | grep -oE '[0-9]+' | sed "q;d")"
+echo $start_end
+}
```
Comment (pandusonu2) [inline downloader.sh:11]: Proper indentation please.

**Review by pandusonu2 — COMMENTED** [review]
The function is wrong, it requires two inputs.
seq <number>q; d
Has variable number input.

File: downloader.sh (hunk 1 above)
Comment (hasan356) [reply downloader.sh:11]: In sed q:d i hv to change nly q part??

Comment (pandusonu2) [reply downloader.sh:11]: If you check the use of the function in downloader.sh, you will see its 
seq "11q;d" or seq "12q;d
The value before the q also changes, that will be second argument to the function

**Comment by pandusonu2** [comment]
Thank you for your contribution :)
While I check the changes, I need you to change the commit message to the following format:
"Fixed #<issue number>: commit message"
I also need you to merge both the commits into a single one.

**Comment by pandusonu2** [comment]
@hasan356 Why did you close the PR and delete the branch?

**Pull request closed without merging**